pennylane/_qubit_device.py

```diff
@@ -1041,7 +1299,8 @@ def expval(self, observable, shot_range=None, bin_size=None):
 
         # estimate the ev
         samples = self.sample(observable, shot_range=shot_range, bin_size=bin_size)
-        return np.squeeze(np.mean(samples, axis=0))
+        res = np.mean(samples, axis=0)
```

Comment:
Oh here some squeezing was removed, was it intentional?

Reply:
Yes, but I'll revert. It does seem like it doesn't make a difference. Yet, I think it will be good to do another sweep of the pipeline here once we've made the new return types the default, to make sure we're not doing stuff like this for no grounded reason.